arc/data/k40_hfs_data.csv

```diff
@@ -1,4 +1,5 @@
 n,l,j,"Hyperfine Magnetic Dipole Constant, A(Hz)","Hyperfine Magnetic Quadrupole Constant, B(Hz)","error A (Hz)","error B (Hz)","theory = 1, experiment = 0",comment,source,DOI
 4,0,0.5,-285730800.00,0.00,2400.00,0.00,0,table IX,"E. Arimondo et. al, Rev. Mod. Phys. 49, 31 (1977)",10.1103/RevModPhys.49.31
+4,1,0.5,-34523000.00,0.00,250.00,0.00,0,table III,"S. Falke et. al, Phys. Rev. A 74 (2006)",10.1103/PhysRevA.74.032503
```

Comment:
This looks good, just the error should be 25000 Hz (instead of 250 Hz) if I read Table III correctly. I will merge to subbranch and fix it, before preparing new version.